src/coreclr/jit/ee_il_dll.cpp

```diff
@@ -1430,3 +1395,14 @@ unsigned Compiler::eeTryGetClassSize(CORINFO_CLASS_HANDLE clsHnd)
 }
 
 #endif // !DEBUG
+
+//------------------------------------------------------------------------
+// eeGetJitVectorInfo: Gets the vector information for the JIT
+//
+// Arguments:
+//    jitVectorInfoPtr -- pointer to the struct that recieves the vector info
+//
+void Compiler::eeGetJitVectorInfo(CORJIT_VECTOR_INFO* jitVectorInfoPtr)
+{
+    info.compCompHnd->getJitVectorInfo(jitVectorInfoPtr);
```

Comment:
Would it be useful to add some asserts here about what values we expect in the CORJIT_VECTOR_INFO returned by the VM? E.g., only 128/256/512? Or based on the arch? In Release, should we fall back to reasonable default values if the numbers aren't what we expect? (Or we could just noway_assert)